Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -25,7 +25,7 @@
 namespace DiscIO
 {
 static u32 ComputeNameSize(const File::FSTEntry& parent_entry);
-static std::string ASCIIToLowercase(std::string str);
+static std::string ASCIIToUppercase(std::string str);
```

Comment:
By simpler, I meant using a standard comparison function (but I have no guaranty that such a function exists and does the same job on every platforms, though) rather than the cost of creating and translating strings.

The only issue I can see is that most of the time these functions return the distance between characters and the distance won't be the same. However, for such a sorting algorithm the distance doesn't matter, so feel free to ignore my previous comments if you think that will fix the issue. My concerns were more about cases we might miss (like strverscmp) than that way of fixing the issue actually.